.drone.yml

```diff
@@ -3,10 +3,47 @@ workspace:
   path: src/github.com/drone-plugins/drone-s3-cache
 
 pipeline:
-  build:
-    image: golang:1.7
-    environment:
-      - CGO_ENABLED=0
+  test:
+    image: golang:1.9
     commands:
+      - go vet
       - go test -cover -coverprofile=coverage.out
-      - go build -ldflags "-s -w -X main.revision=$(git rev-parse HEAD)" -a
+
+  build:
+    image: golang:1.9
+    commands: sh .drone.sh
+    when:
+      branch: master
```

Comment:
IMHO it should always build the binaries just to make sure it can be built for all architectures

Reply:
@tboerger Fixed.